Vault-Deployment/values/otlp.yaml

```diff
@@ -0,0 +1,25 @@
+---
+mode: "deployment"
+replicaCount: 1
+config:
+  receivers:
+    tcplog:
+      listen_address: "0.0.0.0:54525"
+  exporters:
+    loki:
+      endpoint: "http://loki.loki.svc.cluster.local:3100/loki/api/v1/push"
+      headers:
+      # Send everything to the vault-audit tenant_id
+      # https://grafana.com/docs/loki/latest/operations/multi-tenancy
+        "X-Scope-OrgID": vault-audit
```

Comment:
This means OTLP is sending everything it receives to the vault audit tenant. 

This is where it gets tricky:

Either we have one tenant for all audit logs, but then we need to label the logs accordingly to distinguish between apps, or we send everything to vault audit dedicated tenant, but need a dedicated otlp for it or anything sending to otlp will land in vault audit tenant